src/refactor_nrepl/find/find_macros.clj

```diff
@@ -51,6 +51,7 @@
   (util/with-additional-ex-data [:file (.getAbsolutePath f)]
     (with-open [file-rdr (FileReader. f)]
       (binding [*ns* (or (core/path->namespace :no-error f) *ns*)
+                reader/*read-eval* false
```

Comment:
I'm not sure about this one. Do we really need to evaluate analyzed expressions?

Reply:
> What would be the disadvantage of switching evaluation off?

Taking time out of your life to figure out if it's the correct thing to do or not 🤣 

Don't think this will make any difference in this case.

Reply:
> What would be the disadvantage of switching evaluation off?

Taking time out of your life to figure out if this is correct or not? 🤣 

I don't think this will make any difference in this situation.  People never use the reader to eval anyway, so even if this is the wrong thing to do nobody will be affected.